### ISSUE TITLE
Logout functionality

### ISSUE THREAD

**Issue by dylankolson** [acesinc/bullpen-center#41]
there should be a button to allow users to logout of the system

**Comment by TheAndyStumpf** [comment]
Agreed.  OZP didn't include one b/c the environment it was designed for didn't require it.  But in this environment, it is.
